src/verinfast/utils/utils.py

```diff
@@ -131,14 +131,17 @@ def __init__(self, path: str, debug: bool = False):
         self.debug = debug
 
     def log(self, msg, tag=None, display=False, timestamp=True):
-        if tag is None:
+        if tag is None and timestamp:
```

Comment:
This prevents logging the source of the issue without a timestamp. Is that intentional?

Reply:
It is. This was the cause of the mystery long source in my "pretty" log. My theory was a log is either "pretty" (so no source or timestamp) or "full." I was also trying to get this PR done in 10 minutes yesterday to hopefully release yesterday. Now that we are moving it back we could do a tiny bit more.